tests/foreman/cli/test_computeresource_azurerm.py

```diff
@@ -127,7 +135,16 @@ def test_positive_crud_azurerm_cr(self, module_org, module_location, azurerm_set
 
     @upgrade
     @tier2
-    @parametrize("image", [AZURERM_RHEL7_FT_IMG_URN, AZURERM_RHEL7_UD_IMG_URN])
+    @parametrize(
```

Comment:
Use it from the actual library i.e pytest. The local caching of parametrizing isn't giving any additional functionality.

```suggestion
    @pytest.mark.parametrize(
```

Reply:
its decorator of `pytest.mark.parametrize` , not sure if that make it run differently?

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/decorators/__init__.py#L32